[EDU-3856] unbind a service (#654)
aziontech/docs: src/content/docs/en/pages/deploy-journey/unbind-service/unbind-service.mdx

---
title: Unbind an edge service
description: aaa
meta_tags: deploy, orchestrator
namespace: docs_deploy_unbind_service
menu_namespace: deployMenu
og_image: ''
permalink: /documentation/products/guides/deploy/unbind-service/
---

import Button from '~/components/Button.astro'

You can unbind an edge service from an edge node using:

> - [Real-Time Manager (RTM)](/en/documentation/products/guides/how-to-access-rtm/)
> - [Azion API](https://api.azion.com/#32f34add-9e9f-4b25-94c9-3cee4c266a23)

Reviewer: ```suggestion
- [Real-Time Manager (RTM)](#via-rtm)
- [Azion API](#via-api)
```

---

## Via RTM 

> 1. [Access RTM](/en/documentation/products/guides/access-rtm/).
> 2. On the upper-left corner of the page, open the **Products menu**, represented by three horizontal lines, and then select **Edge Nodes**.
> 3. Select the edge node to which you want to unbind a service.
> 4. Go to the **Services** tab.
> 6. Hover the service on the list and click the trash can icon.

Reviewer: ```suggestion
1. [Access RTM](/en/documentation/products/guides/how-to-access-rtm/) > **Edge Nodes**.
2. Select the edge node to which you want to unbind a service.
3. Go to the **Services** tab.
4. Hover over the service on the list and click the trash can icon.
```

Reviewer: talvez vale adicionar o passo 5 para explicar o que acontece/fazer logo de clicar o trash icon

> 6. Hover the service on the list and click the trash can icon.

Author: ```suggestion
5. Hover the service on the list and click the trash can icon.
```

```bash 
curl --location 'https://api.azionapi.net/edge_nodes/' \
--header 'Accept: application/json; version=3' \
--header 'Authorization: Token [TOKEN VALUE]'
```

2. Run the following `LIST` request in your terminal, replacing `[TOKEN VALUE]` with your [personal token](/en/documentation/products/guides/personal-tokens/) and `:id` with the ID of the edge node, and retrieve the ID of the relation between the edge node and the chosen service:

```bash
curl --location 'https://api.azionapi.net/edge_nodes/:id/services' \
--header 'Accept: application/json; version=3' \
--header 'Authorization: Token [TOKEN VALUE]'
```

3. Run the following `DELETE` request in your terminal, replacing `[TOKEN VALUE]` with your [personal token](/en/documentation/products/guides/personal-tokens/), `:node_id` with the ID of the edge node and `:bind_id` with the ID of the relation between edge node and edge service:

```bash
curl --location --request DELETE 'https://api.azionapi.net/edge_nodes/:node_id/services/:bind_id' \
--header 'Accept: application/json; version=3' \
--header 'Authorization: Token [TOKEN VALUE]'
```

> 

Reviewer: ```suggestion
```
